src/App.tsx

```diff
@@ -3,33 +3,56 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  inputHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      query: event.target.value,
+    });
+  };
+
+  filterMovies = (movies: Movie[]) => {
+    this.state.query.toLowerCase();
```

Comment:
you need to set it to some variable, now it not changes the query
```suggestion
    const lowerQuery = this.state.query.toLowerCase();
```